src/prisma/schema.prisma

```diff
@@ -1988,7 +1988,13 @@ model IssuedCredential {
   proposal             Proposal?            @relation(fields: [proposalId], references: [id], onDelete: Cascade)
   rewardApplicationId  String?              @db.Uuid
   rewardApplication    Application?         @relation(fields: [rewardApplicationId], references: [id], onDelete: Cascade)
-  ceramicId            String               @unique
+  schemaId             String?
+  // Fields specific to ceramic
+  ceramicId            String?              @unique
+  ceramicRecord        Json?
+  // Fields specific to on chain credentials
+  onChainAttestationId String?              @unique
```

Comment:
suggestion: "easAttestationId"? or else rename "easChainId" to "onChainChainId" which is funny